oak_attestation_verification/tests/verifier_tests.rs

```diff
@@ -65,6 +66,7 @@ fn create_endorsements() -> Endorsements {
     // Use this for all binaries.
     let tre = TransparentReleaseEndorsement {
         endorsement,
+        attachment: b"".to_vec(),
```

Comment:
I don't think this should be converted from a raw string -- why not just `vec![]` or `Vec::new()`?